definitions/variable/tag_existing_electricitysources.yaml

```
@@ -0,0 +1,95 @@
+- Common Electricity Source:
```

Please keep the file name and tag name identical to common-definitions to simplify keeping the repositories in sync...

file name -> `tag_secondary_electricity_sources.yaml`
```suggestion
- Electricity Source:
```

---

But this is supposed to be the separate tag for the variables we are adding. To avoid the problems with duplicates.